File: src/main/java/com/robohorse/robopojogenerator/action/GeneratePOJOAction.java
package com.robohorse.robopojogenerator.action;

import com.intellij.openapi.actionSystem.AnAction;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.robohorse.robopojogenerator.PluginApplication;
import org.jetbrains.annotations.NotNull;

public class GeneratePOJOAction extends AnAction {
    private final PluginApplication pluginApplication = new PluginApplication();

    @Override
    public void actionPerformed(@NotNull AnActionEvent e) {
        pluginApplication.actionPerformed(e);
    }
}

File: src/main/kotlin/com/robohorse/robopojogenerator/action/GeneratePOJOAction.kt
package com.robohorse.robopojogenerator.action

import com.intellij.openapi.actionSystem.AnAction
import com.intellij.openapi.actionSystem.AnActionEvent
import com.robohorse.robopojogenerator.PluginApplication

internal class GeneratePOJOAction : AnAction() {
    private val pluginApplication = PluginApplication()

    override fun actionPerformed(event: AnActionEvent) = pluginApplication.actionPerformed(event)
}
